plugins/wasm-go/extensions/ext-auth/main.go

```diff
@@ -91,7 +90,7 @@ func checkExtAuth(ctx wrapper.HttpContext, cfg config.ExtAuthConfig, body []byte
 	requestPath := httpServiceConfig.Path
 	if httpServiceConfig.EndpointMode == config.EndpointModeEnvoy {
 		requestMethod = ctx.Method()
-		requestPath, _ = url.JoinPath(httpServiceConfig.PathPrefix, ctx.Path())
+		requestPath = httpServiceConfig.PathPrefix + ctx.Path()
```

Comment:
Whether httpServiceConfig.PathPrefix can end with a /, it is recommended to merge redundant /